src/components/Welcome/Welcome.js

```diff
@@ -1,56 +1,60 @@
 import React from 'react';
 import RecipeCard from '../RecipeCard/RecipeCard.js';
 import './Welcome.css';
-import { getRandomRecipe } from './../../APICalls'
+import { getRandomRecipe, getLatestRecipes } from './../../APICalls'
+import LoginButton from '../LoginButton/LoginButton';
+
 
 const Welcome = () => {
   const  userData = {}
   
   const [randomDrink, setRandomDrink] = React.useState([])
```

Comment:
You'll typically see `useState` or `useEffect` destructured above in the `import` using `import React, { useState, useEffect } from 'react'`. This reduces a little redundancy in saying `React.__`.